cmd/skopeo/layers.go

```diff
@@ -9,13 +14,45 @@ var layersCmd = cli.Command{
 	Name:  "layers",
 	Usage: "get images layers",
 	Action: func(c *cli.Context) error {
-		img, err := parseImage(c)
+		rawSource, err := parseImageSource(c, c.Args()[0])
 		if err != nil {
 			return err
 		}
-		if err := img.LayersCommand(c.Args().Tail()...); err != nil {
+		src := image.FromSource(rawSource)
+		layers := c.Args().Tail()
+		if len(layers) == 0 {
+			ls, err := src.LayerDigests()
+			if err != nil {
+				return err
+			}
+			layers = ls
+		}
+		tmpDir, err := ioutil.TempDir(".", "layers-")
```

Comment:
so, looking at atomic I don't see any problem here since the layers are downloaded in a temp dir here https://github.com/projectatomic/atomic/blob/master/Atomic/util.py#L224 and then the layers are retrieved with https://github.com/projectatomic/atomic/blob/master/Atomic/atomic.py#L1117

I believe an anonymous temp dir in the cwd is fine for atomic and would not bother myself changing something now - if ppl needs to add a destination to `skopeo layers` then will do this otherwise this command will eventually go away as @giuseppe is already looking forward to use `skopeo copy`